gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-bin.zip
```

Comment:
Any reason for not using 5.6.2? I use it and it builds ok.

Reply:
Not really...

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-bin.zip
```